manifest.json

```diff
@@ -15,29 +15,40 @@
   "manifest_version": 1,
   "name": "Netatmo Energy",
   "short_name": "Netatmo ⚡",
-  "version": "0.0.4",
+  "version": "1.0.0",
   "options": {
     "schema": {
       "type": "object",
-      "title": "Login to your Netatmo Smart Home",
+      "title": "Your Netatmo Smart Home app (dev.netatmo.com)",
       "properties": {
```

Comment:
does add to config work even if stuff isn't in the schema?

Reply:
Seems to work, yes :) Not sure how future-proof this is, however I'd rather not expose these field in the add-on settings.

Reply:
.. or not, seems that I somewhere override the config exposed in the add-on settings. Will have a closer look.